### ISSUE TITLE
[bug] Additional resource trees

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14216]
### Summary
In the "Resource Groups", "Import site from HTML" and "Import resources from static files" sections, the additional resource tree does not work correctly:
1) ~There are action icons on the right (update the tree, create a resource, create a child resource), but these icons do not work and **they are not needed there**.~
2) Tips for these icons are displayed incorrectly.

![resource_tree](https://user-images.githubusercontent.com/12523676/50423284-d4220400-086c-11e9-85b9-f689f43aee2c.png)

3) ~In the "Resource Groups" section, you can go to editing the resource by clicking on the resource. In general, **editing is not needed here**, because resources are added to groups by dragging (you can click wrongly).~

### Expected behavior
The functionality of additional trees, in my opinion, should be the same as in the "Top Menu" section, without action icons and tips.

### Environment
MODX 2.7

**Comment by alroniks** [comment]
I am not sure but it seems that this features used rarely. I would remove it from the core at all and replace by a 3rd extra.

**Comment by Ruslan-Aleev** [comment]
@Alroniks Yes, in my opinion, export is not particularly used, but resource groups are useful for permission setting at least.
And how is MODX usage statistics collect? Is statistics collected at all?
May be, for example, enable analytics scripts in final versions 2.x and, with the permission of the site administrator, collect a click map, a conversion map, etc. to understand the functions that are used by MODX-users and are needed in core?

**Comment by JoshuaLuckers** [comment]
I agree with @Ruslan-Aleev, Resource Groups are useful and should, in my opinion, not be removed from the core.

**Comment by alroniks** [comment]
In the issue described 3 points and sometimes it's difficult to answer properly for all of them. I agree that Resource Groups should be in the core, they are part of ACL system. Import can be deleted.

**Comment by patrickatwsrn** [comment]
Point 1 and 2 are still an issue. They are neither correctly displayed nor do they anything at all. 

I would also remove the "edit resource on click" funtionality for elements in the resource tree to the right. It is not needed here and potentially harmful for the user experience.
